internal/provisioner/helm/controllers/bundledeployment_controller.go

```diff
@@ -251,6 +278,16 @@ func (r *BundleDeploymentReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		}),
 		updater.EnsureInstalledName(bundle.GetName()),
 	)
+	if err := r.addDynamicWatch(cl, bd, bundle); err != nil {
```

Comment:
This feels like something the plain provisioner could also hook into?

Reply:
This is checking for the changes in Chart.yaml file.  The plain provisioner is generating the contents of the Chart.yam file on the fly so it's always same.